requirements_llm.txt

```diff
@@ -1,6 +1,7 @@
 sentence-transformers
 faiss-cpu
 
+tokenizers>=0.13.3
```

Comment:
We can put this in main requirements. Also, shouldn't this be handled by transformers itself?

Reply:
I thought so too, but that doesn't seem to be the case